code-challenges.js

```diff
@@ -59,3 +96,9 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+ const numSub = (numOne, numTwo) => {return numOne - numTwo}
+
+ console.log(numSub(number2, number1))
+ console.log(numSub(number3, number4))
+
```

Comment:
How would you account for the two conditions
- 1st number greater than 2nd number
- 1st number less than 2nd number